regression/Makefile

```diff
@@ -82,18 +83,31 @@ $(INITRAMFS)/benchmark:
 $(INITRAMFS_EMPTY_DIRS):
 	@mkdir -p $@
 
-$(INITRAMFS)/opt/syscall_test:
+.PHONY: $(SYSCALL_TEST_DIR)
+$(SYSCALL_TEST_DIR):
 	@make --no-print-directory -C syscall_test
 
+.PHONY: $(INITRAMFS_IMAGE)
+$(INITRAMFS_IMAGE): $(INITRAMFS_FILELIST)
+	@if ! cmp -s $(INITRAMFS_FILELIST) $(INITRAMFS_FILELIST).previous ; then \
+		echo "Generating the initramfs image..."; \
+		cp -f $(INITRAMFS_FILELIST) $(INITRAMFS_FILELIST).previous; \
+		( \
+			cd $(INITRAMFS); \
```

Comment:
I think `cd $(INITRAMFS);` is not necessary here.

Reply:
We need to give `cpio` relative paths, not just the path on the host file system, but the path inside the generated initramfs.

So if we omit `cd $(INITRAM)`, `cpio` has no way to locate the relative path.

```
Generating the initramfs image...
cpio: ./lib: Cannot stat: No such file or directory
cpio: ./lib/x86_64-linux-gnu: Cannot stat: No such file or directory
cpio: ./lib/x86_64-linux-gnu/libc.so.6: Cannot stat: No such file or directory
```

Reply:
OK, thanks for the explanation